Shared/Samples/Run valve isolation trace/RunValveIsolationTraceView.Model.swift

```diff
@@ -273,7 +273,7 @@ extension RunValveIsolationTraceView {
                 .domainNetwork(named: "Pipeline")?
                 .tier(named: "Pipe Distribution System")?
                 .defaultTraceConfiguration
-            else { return nil }
+            else { fatalError("Utility network does not have a default trace configuration.") }
```

Comment:
I guess I meant you could simply force unwrap `configuration`. But I'm not sure if that's the convention in samples.

Reply:
Also this should now return a non-optional with your change.

Reply:
Force unwrap crash and fatalError are both acceptable for a programmer error.